data/root/root.file_list

```diff
@@ -359,7 +359,11 @@ cracklib: nodeps
   x etc/ssh /lib
   x ../rescue/mount-rootfs-and-do-chroot.sh /bin
 
-openssl:
+if exists(openssl, /usr/share/ssl)
+  openssl:
```

Comment:
(note: https://github.com/openSUSE/installation-images/blob/master/doc/files.md#conditional-sections explains the condition syntax)

Um, could we use a simple `openssl*` here? That would match both `openssl-*` and `openssl`. Or not?